frontend/fingertips-frontend/components/organisms/LineChart/lineChartHelpers.ts

```diff
@@ -192,6 +194,9 @@ export function generateStandardLineChartOptions(
     ...lineChartDefaultOptions,
     yAxis: {
       ...lineChartDefaultOptions.yAxis,
+      labels: {
+        formatter: optionalParams?.yAxisLabelFormatter,
+      },
```

Comment:
This is overwriting the font sizing set in `lineChartDefaultOptions` for the Y axis.

You'll need to do something like:
```suggestion
      labels: {
        ......lineChartDefaultOptions.yAxis.labels
        formatter: optionalParams?.yAxisLabelFormatter,
      },
```

Reply:
Yes. I had meant to return to this as the type for yAxis made this awkward